### DISCUSSION TITLE
How to suppress page-level bibliography?

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#485] [Get Help] [answered]
Hi everyone! I'm working on two books for the Yale University Art Gallery, and both of them will have footnotes that include short-form citations. When I add the citations with the q-cite shortcode, the full citation is also added at the bottom of the page as a page-level bibliography. How can I make this page-level bibliography disappear? We only want the complete citations to appear on the bibliography page at the end of the book (and in the hover pop-up window on the page). 

If this is an easy answer, it could be helpful to add this to the documentation page on citations and bibliographies. Or is the answer there, and I'm just not seeing it? I hope not! I've looked for it several times. 

Thank you!

**Comment by Erin-Cecele** [comment, marked as answer]
Hi @zsofiaj - This has come up for us a few times at Getty too, so great question! There are two ways you can approach removing the page-level biblio. The first is to use custom CSS to hide the biblio:

```
.quire-page__content .container .content .backmatter {
  display: none;  
}
```

Or, if you want to actually remove it, rather than just hide it, you would need to delete it from the page template. Read more about overriding page templates here: https://quire.getty.edu/documentation/styles-customization/#override-theme-templates. 

We are working on a way to simplify suppressing the page-level bios in the new 11ty version of Quire so stay tuned.

**Reply by Erin-Cecele** [reply]
Hi @zsofiaj, just a note about my suggestion above. Using the selector above that includes `backmatter` will suppress anything on any page inside a backmatter class` <div>`. So use with caution.

**Reply by zsofiaj** [reply]
Thanks so much, @Erin-Cecele! I can't use the custom CSS route because we do have other backmatter elements on the page that we want to retain (published references, exhibition history, and things like that). I followed the instructions on overriding page templates, and that worked! I created a new layouts folder in my main directory, copied over the entry subdirectory with the single.html file, and I deleted the line about the page-level biblio. Thanks again!

**Reply by Erin-Cecele** [reply]
@zsofiaj - Excellent. Thank you for updating us!
